index.js

```diff
@@ -59,10 +79,10 @@ app.post('/login',(request, response) => {
         if(request.body.username == username && request.body.password == password) {
             request.session.login = username;
             console.log(users[i]);
-            return response.render('login-success');
+            return response.render('about');
         }
     }
-    return response.render('login-failed');
+    return response.render('loginform');
```

Comment:
this should be `login-failed` and you'll have to create a `login-failed.ejs` to handle the failed logins.